### ISSUE TITLE
No error is shown when publish fails

### ISSUE THREAD

**Issue by IjzerenHein** [tiaanduplessis/pkg-rename#9]
Hi, first of all, thanks for this useful package!

I've noticed an issue with it. I tried to rename a package (`react-navigation-sharedelement` -> `react-navigation-shared-element`), but this didn't complete succesfully.
I used this command after renaming the name in package.json:

```sh
$ pkg-rename react-navigation-sharedelement --publish
```

And this was the output:
```sh
Successfully deprecated <=0.4.5 versions of react-navigation-sharedelement.
```

Unfortunately the publish did not happen or complete succesfully. When publishing manually I get the following NPM error `403 Forbidden - PUT https://registry.npmjs.org/react-navigation-shared-element - Package name too similar to existing packages; try renaming your package to '@ijzerenhein/react-navigation-shared-element' and publishing with 'npm publish --access=public' instead`, which explains the problem.

So in short, when publish fails it should report the error, otherwise it gives you the impression that all went well.

cheers!
